app/core/myeclpay/endpoints_myeclpay.py

```diff
@@ -1054,17 +1054,14 @@ async def register_user(
 
 @router.get(
     "/myeclpay/tos",
+    response_class=FileResponse,
```

Comment:
This will force changing the way the front handles this response. Can't you just open, read and return the content of the file instead of the file itself ?